packages/core-p2p/src/transaction-broadcaster.test.ts

```diff
@@ -0,0 +1,60 @@
+import { Container } from "@arkecosystem/core-kernel";
```

Comment:
This file should go to `__tests__/unit/core-p2p/transaction-broadcaster.test.ts`

Reply:
Moved